### DISCUSSION TITLE
how set path of flatnote at startup

### DISCUSSION THREAD

**Discussion by gandhi404** [mediagis/nominatim-docker#242] [Q&A] [answered]
Hello,
is it possible to set the path of the flatnote files in the docker run command with -v or another parameter? Or do I have to do this in the env file via a * .sh when starting?

Many thanks and best regards

**Comment by philipkozeny** [comment, marked as answer]
Do you mean on your host machine, or inside the docker container? Host machine can be achieved by mounting the correct directory.

**Reply by gandhi404** [reply]
sorry my mistake!
